src/nifs/protogalaxy/poly/mod.rs

```diff
@@ -304,22 +304,39 @@ pub(crate) fn compute_G<C: CurveAffine>(
     }
 }
 
+pub(crate) struct Challenges<F: PrimeField> {
+    alpha: F,
+    beta: F,
+    delta: F,
+}
+
+impl<F: PrimeField> Challenges<F> {
+    fn beta_stroke(&self) -> F {
+        self.beta + (self.alpha * self.delta)
```

Comment:
beta* is a vector of:

[(beta+alpha*delta), (beta^2+alpha*delta^2),...]

only the first item in the vector is not enough

Reply:
Seems this same detail is missing in `compute_G`, I'll fix it now, thanks!